src/gleam/dict.gleam

```diff
@@ -474,13 +474,63 @@ pub fn upsert(
   |> insert(dict, key, _)
 }
 
-@deprecated("This function has been renamed to `upsert`")
+/// Returns a dict where the caller attempts to change a key-value pair in the
```

Comment:
I am not happy with this explanation.

Reply:
Any better ideas?

Reply:
Perhaps something more concise?

> Updates the value in a dict for a given key using a given function.
>
> If there already was a value in the dict for the key then it is passed into the function.
>
> If a value is returned from the function then it will be used as the new value in the dict. If no value is returned then any previous value is removed.

Reply:
This shows an example where I cannot see that the function description above matches. Because it really depends on the change callback/function what happens here.

https://github.com/gleam-lang/stdlib/pull/659/files#diff-99caf068f3c916909bcf3231d839db8a8afdebb65dc04d0353eea53071729264R265-R278